### PR TITLE
Feature/40010

### PR DESCRIPTION
**Pull request by abhipunj0812**
Create a file Database.ts and installed Mongodb and checked its connection.

### CODE REVIEW

File: src/libs/Database.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+               console.log(err);
+               reject(err);
+               return;
+           } else {
+           resolve(undefined);
+           }
+        });
+    }); }
+    static disconnect() {
+        console.log('inside disconnect method');
```
Comment (rahulsainidev) [inline src/libs/Database.ts]: Add code to disconnect the connection from mongodb.

File: src/Server.ts [hunk 2]
```diff
@@ -27,6 +28,10 @@ class Server {
     }
     run() {
         const { app, config: { PORT } } = this;
+        Database.open('mongodb://localhost:27017/express-training')
```
Comment (rahulsainidev) [inline src/Server.ts]: Import the mongo URL from config instead of hardcoding here.

**Review by rahulsainidev — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -24,10 +24,15 @@
...
   "dependencies": {
+    "@types/mongoose": "^5.7.37",
     "body-parser": "^1.19.0",
     "commitizen": "^4.2.2",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "jsonwebtoken": "^8.5.1",
+    "mongodb": "^3.6.2",
+    "mongoose": "^5.10.13",
+    "secret-key": "^1.4.3",
```
Comment (rahulsainidev) [inline package.json]: Why this package ("secret-key") is needed, If the unused remove this

Comment (abhipunj0812) [reply package.json]: ok sir

**Review by rahulsainidev — APPROVED** [review]

**Pull request merged**